source/wp-content/themes/wporg-developer-2023/src/style/style.scss

```diff
@@ -120,20 +134,19 @@ body[class] {
 }
 
 @media (max-width: 781px) {
-	.wp-block-column[style*="border-right-"] {
-		border-bottom: 1px solid var(--wp--preset--color--light-grey-1);
-		border-right: none !important;
-	}
+	.home-section-docs-api {
+		padding-left: 0 !important;
+		padding-right: 0 !important;
 
-	.wp-block-column[style*="border-right-"][style*="padding-right"] {
```

Comment:
I'm unsure whether this pattern is used in other pages, so maybe the styles shouldn't be removed.

For this particular section (Documentation & API Reference), once the horizontal padding is removed from the columns to allow the layout to work in both directions, the border styling breaks.

IMO this changeset is simpler (no negative margin) and less brittle, but it's also less reusable. Open to discussion.